dev_env/core/runner.py

```diff
@@ -0,0 +1,104 @@
+import json
+import os
+import re
+import shutil
+import time
+
+from core.prompts import Prompt, Colors
+from core.utils import Runner
+
+
+class DockerCompose(Runner):
+    REQUIRED_PORTS = [8000]
+
+    def generate_config(self):
+        config = {
+            "admins": [],
+            "branding": "",
+            "db": "postgres://postgres:PASSWORD@postgres_dev:5432/govready_q",
+            "debug": True,
+            "enable_toolbar": False,
+            "email": {},
+            "govready_cms_api_auth": "",
+            "govready-url": "http://localhost:8000",
+            "mailgun_api_key": "",
+            "memcached": False,
+            "secret-key": self.create_secret(),
+            "gr-pdf-generator": "wkhtmltopdf",
+            "gr-img-generator": "wkhtmltopdf",
+            "single-organization": "",
+            "static": "static_root",
+            "syslog": "",
+            "trust-user-authentication-headers": {}
```

Comment:
```suggestion
            "syslog": "",
            "test_visible": False
            "trust-user-authentication-headers": {}
```
missed test_visible key

Reply:
added